cluster/gce/gci/configure-helper.sh

```diff
@@ -2364,6 +2367,8 @@ EOF
     fi
     prepare-kube-proxy-manifest-variables "$src_dir/kube-proxy/kube-proxy-ds.yaml"
     setup-addon-manifests "addons" "kube-proxy"
+  elif [[ -z "${KUBE_PROXY_TOKEN:-}" ]]; then
```

Comment:
This is a weird thing to pass to the master, since we don't necessarily run kube-proxy there.

Reply:
it needs to be passed to the master because it needs to be injected into `known_tokens.csv`: https://github.com/kubernetes/kubernetes/blob/5863e53bda3b81382a89075db07ab958e7f8c16d/cluster/gce/gci/configure-helper.sh#L642-L644